chore: remove namedExports option, add nodePolyFills

kjappelbaum opened the pull request:
-The namedExports option from @rollup/plugin-commonjs is deprecated. Named exports are now handled automatically.
https://github.com/rollup/plugins/pull/410

targos commented:
The named exports removal sgtm. Can you show me a project where nodePolyFills is needed?

Commit:
chore: remove namedExports option

-The namedExports option from @rollup/plugin-commonjs is deprecated. Named exports are now handled automatically.

kjappelbaum commented:
I think i'll need `nodePolyFills` here https://github.com/cheminfo/xrd-tools. 

But I agree with you that it is probably better to keep this out of this PR and do it later when needed.  Just removed this part from the PR.
